src/UserWarning.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
why do you need this

Reply:
Because If I do not comment line it is throw me an error, and all file everywhere marked red color and can not deploy. I can not fix ESLint by other way